tests/angular_devkit/build_angular/hello-world-app/src/tsconfig.server.json

```diff
@@ -2,6 +2,7 @@
   "extends": "./tsconfig.app.json",
   "compilerOptions": {
     "outDir": "../dist-server",
+    "baseUrl": "./",
```

Comment:
This doesn’t seem right, the badeUrl should only be defined in the workspace config, otherwise paths for local workspace libraries won’t work.

Reply:
I didn't really change that. Those diffs are a bit messed up but what I really did was `hello-world-app` -> `hello-world-app-ve` and then `hello-world-app-ivy` -> `hello-world-app`. So that was there before. It might be wrong still ofc.

Reply:
I didn't really change that. Those diffs are a bit messed up but what I really did was `hello-world-app` -> `hello-world-app-ve` and then `hello-world-app-ivy` -> `hello-world-app`. So that was there before. It might be wrong still ofc.